Formula/brev-cli.rb

```diff
@@ -1,18 +1,21 @@
 class BrevCli < Formula
   desc "CLI tool for managing workspaces provided by brev.dev"
   homepage "https://docs.brev.dev"
-
   url "https://github.com/brevdev/brev-cli/archive/refs/tags/v0.6.12.tar.gz"
   license "MIT"
-
   depends_on "go" => :build
 
   def install
-    system "go", "build", "-o", "brev",  "-ldflags", "\"-X github.com/brevdev/brev-cli/pkg/cmd/version.Version=v.0.6.12\""
+    system "go",
+           "build",
+           "-o",
+           "brev",
+           "-ldflags",
+           "\"-X github.com/brevdev/brev-cli/pkg/cmd/version.Version=v.0.6.12\""
     bin.install "brev"
```

Comment:
```suggestion
    system "go", "build", *std_go_args(ldflags: "-X github.com/brevdev/brev-cli/pkg/cmd/version.Version=v#{version}")
```

Reply:
ty!

Reply:
will this work with the `-o` flag as well? I'm not sure how to include this with `std_go_args`

Reply:
Yes, just append `-o`. Though if your tool is called `brev`, I wonder if the formula should be called `brev` too instead of `brev-cli`. That way we also don't need to add `-o` here.

Reply:
Can also try `*std_go_args(output: bin/"brev", ldflags: "-X github.com/brevdev/brev-cli/pkg/cmd/version.Version=v#{version}")`. See the [`cfssl`](https://github.com/Homebrew/homebrew-core/blob/HEAD/Formula/cfssl.rb#L25) formula for another example.

Reply:
the `-o`  is needed i think regardless of the formula name, but i think it does make sens e to change the formula name to brev, I'll test it both ways.